spec/integration/api/v1/vm_spec.rb

```diff
@@ -56,32 +57,37 @@ def app()
     end
 
     describe 'POST /vm' do
+
+      let(:socket) { double('socket') }
       it 'returns a single VM' do
-        create_ready_vm 'pool1', 'abcdefghijklmnop'
+        create_ready_vm 'pool1', vmname
+        allow_any_instance_of(Vmpooler::API::Helpers).to receive(:open_socket).and_return(socket)
```

Comment:
I could not find a way to use allow to stub the return value of the open_socket method without using `allow_any_instance_of`. I suspect there's something I don't understand with where sinatra is loading the helpers, and that if I were to understand that better there is likely another way to get at this. I'm all for a cleaner solution, but after several days of banging my head against this I did not find one.